components/Tabs/src/MDCTabBar.h

```diff
@@ -112,7 +137,7 @@ IB_DESIGNABLE
 
  The default value is based on the position and is recommended for most applications.
  */
-@property(nonatomic) IBInspectable BOOL displaysUppercaseTitles;
+@property(nonatomic) IBInspectable BOOL displaysUppercaseTitles UI_APPEARANCE_SELECTOR;
```

Comment:
I'm not sure why this would be UIAppearance proxied if the `alignment` property is not. If a client wants to "theme" provide a custom "uppercase for this alignment" behavior, then should they be able to set on a per-alignment basis?

Reply:
Hmm that's a good point, that is inconsistent with UIAppearance.

However, `position` for tab bars is becoming less relevant because `MDCBottomNavigation` is taking over responsibility for the bottom nav use case. I would prefer not to add new position-based APIs that would just be removed later.

The intent here is to reflect a change in our support of this property. Previously, it was a rarely-used release valve for when automatic upcasing was undesirable. (Cases were basically "Default" or "Don't upcase") This was the motivation behind the minimal API before: rarely used, easy to set to NO if you needed it.

Now it's an officially-supported styling parameter: an app may want all of its tab bars to be non-upcased as part of its style.

The cleanest way I can come up with to update this API to accommodate UIAppearance and not bake in position-dependence is to generalize this to an enum.